### PR TITLE
EUI-4321 - Moving find-person files

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
FindPersonComponent
FindAPersonService
Models

Resolves https://tools.hmcts.net/jira/browse/EUI-4321

Notes:
* Moving find-person files to common-lib
* Relocating FindPersonComponent, FindAPersonService
* Adding in Person, SearchOptions

### CODE REVIEW

**Review by rayliangatos — APPROVED** [review]
approved but I have some further changes on the find person component to support other search criterial, please hold on the merge and let me finish my pr on rpx-xui-webapp.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "1.0.1-module-fixes.8",
+  "version": "1.0.1-find-person",
```
Comment (rayliangatos) [inline package.json:3]: it also needs to update the verson on  \projects\exui-common-lib\package.json

**Review by rayliangatos — COMMENTED** [review]
hold on the pr until the latest version on MC is merged

**Review by kuda1992 — APPROVED** [review]
Well done looks good @connorpgpmcelroy.

**Pull request merged**